internal/services/appconfiguration/app_configuration_feature_resource.go

```diff
@@ -175,8 +186,8 @@ func (k FeatureResource) Create() sdk.ResourceFunc {
 
 			appCfgFeatureResourceID := parse.AppConfigurationFeatureId{
 				ConfigurationStoreId: model.ConfigurationStoreId,
-				Name:                 model.Name,
-				Label:                model.Label,
+				Name:                 url.QueryEscape(model.Name),
+				Label:                url.QueryEscape(model.Label),
```

Comment:
the API don't accept the value if its not escaped? the error was in parsing?

Reply:
the API does accept unescaped value, escaping here is to work for `appCfgFeatureResourceID.ID()`.  Do you mean we only need to patch the parsing process?

Reply:
ok, modified. I was worried using a regexp in parsing process maybe not stable as we except so I used escape to do this patch.